blocks/init/src/Blocks/custom/lottie/manifest.json

```diff
@@ -0,0 +1,233 @@
+{
+	"$schema": "https://raw.githubusercontent.com/infinum/eightshift-frontend-libs/develop/schemas/block.json",
+	"blockName": "lottie",
+	"title": "Lottie",
+	"description": "Lottie block with custom settings.",
+	"category": "eightshift",
+	"blockClass": "lottie",
+	"blockJsClass": "js-lottie",
```

Comment:
block has its own blockJsClass by default either use that or `js-block-lottie` or create a new selector. Aslo for `blockClass`